liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/applications/InstallAppMojoSupport.java

```diff
@@ -216,11 +216,11 @@ private void addLibraryFromM2(Artifact library, LooseConfigData config) throws E
     
     private void validateAppConfig(String fileName) throws Exception {
```

Comment:
Need to handle the case when appDirectory is set but the value is not "apps" or "dropins".